recipes/qcbor/all/patches/1.4-0001-fix-cmake.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index e4aa83f..4878a31 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -58,10 +58,14 @@ target_compile_definitions(qcbor
+         $<$<BOOL:${QCBOR_OPT_DISABLE_FLOAT_ALL}>:USEFULBUF_DISABLE_ALL_FLOAT>
+ )
+ 
+-if (BUILD_SHARED_LIBS)
++if (0)
+     target_compile_options(qcbor PRIVATE -Os -fPIC)
+ endif()
+ 
++set_target_properties(qcbor PROPERTIES
++    WINDOWS_EXPORT_ALL_SYMBOLS ON
++)
++
+ # The math library is needed for floating-point support.
+ # To avoid need for it #define QCBOR_DISABLE_FLOAT_HW_USE
+ if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
+@@ -88,6 +92,9 @@ set_target_properties(
+ include(GNUInstallDirs)
+ install(
+     TARGETS qcbor
++    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+     PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/qcbor"
```

Comment:
The include folder is different in the new version compared to the old one, it's a bit suprising that no recipe change was needed 🤔 any insight into this @toge?

Reply:
@AbrilRBS 
qcbor/1.4.0 seems to define public header files in CMakeLists.txt. 

https://github.com/laurencelundblade/QCBOR/commit/3e0bdc5b5d22192b5a6a8ce14269f77cd6b2bdc9